packages-ext/providers-cloud/src/providers/deepinfra.ts

```diff
@@ -0,0 +1,41 @@
+import { createChatProvider, createEmbedProvider, createModelProvider, merge } from '@xsai-ext/shared-providers'
+
+/**
+ * [DeepInfra](https://deepinfra.com/) provider
+ *
+ * @see {@link https://deepinfra.com/pricing} and {@link https://deepinfra.com/docs/openai_api}
+ */
```

Comment:
```suggestion
/** @see {@link https://deepinfra.com} */
```

Reply:
I still think we should have the title included for user to search and reference with.

Reply:
Especially for fireworks, having `Fireworks.ai` with link will be easier for user to find and navigate.